python/setup.py

```diff
@@ -118,23 +126,34 @@ def symlink_keyvi():
     else:
         yield None, None
 
+
 @run_once
-def cmake_configure(build_path, build_type, zlib_root, additional_compile_flags):
+def cmake_configure(build_path, build_type, zlib_root, additional_compile_flags, osx_architectures=None, cmake_prefix_path=None):
     # needed for shared library
     CMAKE_CXX_FLAGS = additional_compile_flags + ' -fPIC'
 
     cmake_configure_cmd = 'mkdir -p {}'.format(build_path)
     cmake_configure_cmd += ' && cd {}'.format(build_path)
     cmake_configure_cmd += ' && cmake' \
-                        ' -D CMAKE_CXX_FLAGS="{CXX_FLAGS}"'.format(CXX_FLAGS=CMAKE_CXX_FLAGS.strip())
-    cmake_configure_cmd +=  ' -D CMAKE_BUILD_TYPE={BUILD_TYPE}'.format(BUILD_TYPE=build_type)
+        ' -D CMAKE_CXX_FLAGS="{CXX_FLAGS}"'.format(
+            CXX_FLAGS=CMAKE_CXX_FLAGS.strip())
+    cmake_configure_cmd += ' -D CMAKE_BUILD_TYPE={BUILD_TYPE}'.format(
+        BUILD_TYPE=build_type)
```

Comment:
```suggestion
    cmake_configure_cmd += f' -D CMAKE_BUILD_TYPE={build_type}'
```

Maybe better we use f-strings if you want to enforce 80 chars line limit ?